daemon/attach.go

```diff
@@ -60,6 +60,7 @@ func (daemon *Daemon) ContainerAttach(prefixOrName string, c *backend.ContainerA
 
 	if err := daemon.containerAttach(container, stdin, stdout, stderr, c.Logs, c.Stream, keys); err != nil {
 		fmt.Fprintf(outStream, "Error attaching: %s\n", err)
+		return err
```

Comment:
When it comes to here, the connection between client and server is already hijacked(by `GetStreams` https://github.com/docker/docker/pull/26069/files?diff=unified#diff-fd2a8512ddf5befe19ef86c84991419dR37) , the connection is a stream, `return err`  here will not return the err to the client.  I think `return err` is not necessary here, but other maintainers may have different thoughts.  :)